python/lsst/task/base/__init__.py

```diff
@@ -1,6 +1,7 @@
 from .version import *
 from .struct import *
 from .task import *
+from .task_metadata import *
```

Comment:
Does TaskMetadata have to be made available externally?

Reply:
If we can eliminate all non-dict deprecated methods, TaskMetadata is not needed. But for now, it should be public, because its methods are used externally.

Reply:
I don't understand this. Previously we didn't export anything because it was a PropertySet and things worked fine. There can't be any external user that needs TaskMetadata because it hasn't previously existed.